### PR TITLE
Fix reviewers on pull_request.opened

### PR DESCRIPTION
**Pull request by gimenete**
For some reason now (and not last week) when there's a `pull_request.opened` event the pull request has its `request_reviewers` and `request_teams` set to `[]` even if you requested somebody to review it when the PR was created. So I'm doing a GET for the PR and I get that information correctly.

This looks like a bug in the GitHub webhook.

### CODE REVIEW

**Comment by gimenete** [comment]
@bkeepers I've been able to get all the information in just one request.

btw, should we change this? https://github.com/integrations/slack/blob/0cfaabaa04fe5db6a4d8c33d5d71ad1760a2f32a/lib/messages/index.js#L2

We can have up to 4 attachments (assignees, labels, milestone, reviewers) sometimes but this variable limits them to a maximum of two.

File: lib/activity/pull-requests.js [hunk 1]
```diff
@@ -21,7 +21,11 @@ async function pullRequestEvent(context, subscription, slack) {
       headers: { accept: 'application/vnd.github.html+json' },
     }))).data;
 
+    // need seperate api request to get reviewers
+    const pr = (await context.github.pullRequests.get(context.issue())).data;
+
     pullRequest = {
+      ...pr,
       ...pullRequest,
```
[the remark this answers is not in the document]
Comment (bkeepers) [reply lib/activity/pull-requests.js]: Yep, that makes sense. I'm just curious if we need the version from the payload. Is there anything in it that is not in API response?

 I think this whole assignment to `pullRequest` can just be replaced with the `pr` object returned from the API.

**Comment by bkeepers** [comment]
> btw, should we change this?

I'd delegate to @wilhelmklopp on that. I think we added this because it just gets too noisy to have all of these fields.

File: lib/activity/pull-requests.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (gimenete) [reply lib/activity/pull-requests.js]: I see. Yeah, probably we can remove the `pullRequest` object completely. I'll take another look.

[the remark this answers is not in the document]
Comment (gimenete) [reply lib/activity/pull-requests.js]: @bkeepers I've just simplified the code following your suggestion. Take a look! :)

**Comment by wilhelmklopp** [comment]
Yeah, keeping it at two fields keeps it to a single line, which in turn keeps the message nice and short. Happy to modify this but overall it feels like we're trending towards finding ways to make messages even shorter (taking up less vertical space) as opposed to more.

**Pull request merged**